pokecli.py

```diff
@@ -353,6 +361,11 @@ def init_config():
     config.action_wait_min = load.get('action_wait_min', 1)
     config.raw_tasks = load.get('tasks', [])
     config.vips = load.get('vips',{})
+    config.map_object_cache_time = load.get('map_object_cache_time')
```

Comment:
I think you should add default here as well?

config.map_object_cache_time = load.get('map_object_cache_time', 5.0)

Otherwise it won't default and an error will reported to the user stating it's out of range.